src/test/java/features/47 Swag Labs Login.feature

```diff
@@ -13,10 +13,13 @@ Feature: 47 Swag Labs Login
   	  And I click the Login Button
   	Then I am redirected to the Sauce Demo Main Page
   	  And I verify the App Logo exists
+  	  And video recording was attached to the report
```

Comment:
See if you can add a @record tag at the top of this scenario and then read it in the @After step in the TestRunner instead of doing a Cucumber step here.